src/features/create-dao/CreateDAOModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, useState } from 'react';
+
+import { BasicModalProps } from '../../lib/types/ui';
+
+import { Modal } from '@zero-tech/zui/components';
+import { CreateDAOForm } from './';
+
+export interface CreateDAOModalProps extends BasicModalProps {
+	domainName: string;
+}
+
+export const CreateDAOModal: FC<CreateDAOModalProps> = ({ domainName, ...props }) => {
+	const [isOpen, setIsOpen] = useState(false);
+
+	const handleClose = (): void => {
```

Comment:
just following on from some guidelines I picked up from the team yesterday, could we change `handleClose` to `onClose`